.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,39 @@
+---
+
+name: Bug Report
+about: Create a bug report to help us improve pandas
+title: "BUG:"
+labels: ""
```

Comment:
I'd say we should use this `label:` feature to automatically tag the issue with the appropriate label. For this template `Bug` and similarly for the others.

https://github.community/t5/How-to-use-Git-and-GitHub/Apply-labels-automatically-at-issue-creation/td-p/10719#

Reply:
Just making sure before I make the changes, the labels for each file should look like this:

---

bug_report: Bug

---

documentation_improvement: Docs

---

feature_request: Enhancement?

---

submit_question: "Usage Question"

Reply:
I was originally on the fence about automatically adding labels because it could make it harder to distinguish what has been triaged vs. what hasn't, since I currently look for the presence of labels to determine if something has been triaged, and automatically adding labels would make it less clear whether a triage has occurred or not.

A potential workaround here is to also automatically add a new "Needs Triage" label and then remove it as part of the triage process.  Or maybe this is only an issue on my part, in which case I can work around it.

Reply:
We don't have a label for feature requests

Reply:
> We don't have a label for feature requests

We typically use the "Enhancement" label for this, so I think what you've proposed above is good.

Reply:
> A potential workaround here is to also automatically add a new "Needs Triage" label and then remove it as part of the triage process.

@jschendel I like that idea, have you seen repositories that do this kind of thing?

Reply:
Yeah, I've seen a couple variants of this: [rapidsai/cudf](https://github.com/rapidsai/cudf/blob/fa525bf1baf0619901de0f356d64baa059448326/.github/ISSUE_TEMPLATE/bug_report.md) automatically adds a generic "? - Needs Triage" label, [scikit-learn/scikit-learn](https://github.com/scikit-learn/scikit-learn/blob/a5dfba94ff34b27848a1094c01c6f6a3dbdf1dae/.github/ISSUE_TEMPLATE/bug_report.md) adds a "Bug: triage" just to bug reports.

Reply:
Numba has a `need triage` label: https://github.com/numba/numba/issues?q=is%3Aissue+is%3Aopen+label%3Aneedtriage

I'd be in favor in autotagging with `<Bug/Doc/Enhancement/...>  + Need Traige`, and then `Need Triage` can be removed once someone has commented (and can be replace with a more specific label like `IO CSV` for example)

I've also been making an effort to tag older issue with one of Bug/Doc/Enhancement/... which will be useful in the future for developing metrics about total number of issues per category.

Reply:
@mroeschke ATM ```Need Triage``` label does not exists.
Should I wait for it to be created? or just edit this pull request to auto-assign the label ```Need Triage``` and the label will be created later.

Reply:
I went ahead and just created a `Needs Triage` label so it should be available now.

Reply:
Thank you @mroeschke